parsec/mca/device/device.c

```diff
@@ -443,23 +440,11 @@ void parsec_devices_print_statistics(parsec_context_t *parsec_context, uint64_t
     parsec_compute_best_unit( total_data_out,     &best_data_out,     &data_out_unit     );
     parsec_compute_best_unit( total_d2d,          &best_d2d,          &d2d_unit          );
 
-    if( 0 == total_required_in ) {
```

Comment:
What if `*_required_*` is 0 (aka the device was not used) ?

Reply:
see above, produces a nan value that we will print.